roles/confluent.variables/defaults/main.yml

```diff
@@ -105,6 +105,9 @@ archive_group: ""
 ### If the installation_method is 'archive' then this will be the base path for the configuration files, otherwise configuration files are in the default /etc locations. For example, configuration files may be placed in `/opt/confluent/etc` using this variable.
 archive_config_base_path: "{{ archive_destination_path }}"
 
+### The configuration prefix to use, by default /etc
```

Comment:
Add something along the lines of "Only valid to customize when installation_method: archive"
Also add that to the rest of these vars